.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Let's make this `**/.DS_Store` so we exclude all & any instances of .DS_Store files

Reply:
Done!